src/hashTable.js

```diff
@@ -0,0 +1,125 @@
+class Node {
+  constructor(key, value) {
+    this.key = key,
+    this.value = value,
+    this.next = null
+  }
+}
+
+export default class HashTable {
+  constructor() {
+    this._length = 0
+    this.valueStore = []
```

Comment:
If you choose to use the `_` naming convention to indicate "private" data/instance variables, you should do so consistently (i.e. `valueStore` should be `_valueStore`).